safenode/src/network/msg/mod.rs

```diff
@@ -33,12 +33,17 @@ impl SwarmDriver {
                     ..
                 } => {
                     trace!("Received request with id: {request_id:?}, req: {request:?}");
-                    self.event_sender
-                        .send(NetworkEvent::RequestReceived {
-                            req: request,
-                            channel: MsgResponder::FromPeer(channel),
-                        })
-                        .await?
+
+                    if let Request::Cmd(Cmd::Replicate(ReplicatedData::Chunk(chunk))) = request {
```

Comment:
Well, if using the normal flow on receiving a Request::Cmd, it will be:
   swarm_driver -> node -> handling -> swarm_driver
Where swarm_driver is calling the `disk_storage` to write the record to disk.
As the record is a chunk, which is bit mem consuming, hence made it short-cut here, to avoid an internal un-necessary trip.

It is just `Request` is a facility for peering messaging to be used, over which we can define which are for the Node-Node communication and which are for SwarmDriver-SwarmDriver communication.